internal/cmd/gocdk/biomeutil.go

```diff
@@ -132,7 +132,7 @@ func tfReadOutput(ctx context.Context, dir string, env []string) (map[string]*tf
 
 // tfOutput describes a single output value.
 type tfOutput struct {
-	Type      string      `json:"type"` // one of "string", "list", or "map"
+	Type      interface{} `json:"type"`
```

Comment:
I don't understand this change. What breaks here?

Reply:
`terraform output -json` doesn't return "map" for the type anymore:

```
  "launch_specifier": {
    "sensitive": false,
    "type": [
      "object",
      {
        "host_port": "number"
      }
    ],
    "value": {
      "host_port": 8080
    }
  }
```

Since we don't actually use it, I just changed it to `interface{}`.

Reply:
Interesting. Let's remove the field since we don't use it. :)

Reply:
OK. Removed `Sensitive` as well, since it's also unused.